app/views/missing/report.php

```diff
@@ -136,7 +136,7 @@
                                         <label for="lname">Last name </label>
                                         <input type="text" class="form-control" placeholder="Enter Last Name" id="lname" name="person[lname]" required /><br>
                                         <label for="dob">Date of birth </label>
-                                        <input type="date" placeholder="Enter your DOB" class="form-control" id="dob" type="date" name="person[dob]" /><br>
+                                        <input type="date" placeholder="Enter your DOB (mm/dd/yyyy)" class="form-control" id="dob" type="date" name="person[dob]" pattern="\d{1,2}/\d{1,2}/\d{4}" /><br>
```

Comment:
The date type is not supported in some browser, and AFAIK, the placeholder and pattern is only effective in those browser. Also, if the format is `mm/dd/yyyy` the MySQL does not directly accepts this as date. So, for these browser the format should be `yyyy-mm-dd` which MySQL accepts as date.

Also, we can also improve regex a bit so that it is somewhat more restrictive about valid dates. (Should be available easily).